### DISCUSSION TITLE
can we add a ttl for match_t?

### DISCUSSION THREAD

**Discussion by zjlooojoe** [Kong/kong#13917] [Help]
match_t is what we get from router. Can we add a ttl for it?

**Comment by zjlooojoe** [comment]
or can we add ttl for router?

**Reply by zjlooojoe** [reply]
what i need is to invalidate the cache "router:version", right? And there is a timer checking the cache "router:version". If it has changed, the worker will rebuild the router. Am i right?

**Comment by Water-Melon** [comment]
Could you elaborate on the background of this requirement?
